definitions/npm/react-i18next_v6.x.x/test_react-i18next_v6.x.x.js

```diff
@@ -50,16 +50,12 @@ class ClassComp extends React.Component<Props> {
 }
 
 // $ExpectError - wrong argument type
-const FlowErrorComp = ({ s, t }: Props) =>
```

Comment:
This became an unused suppression in flow v65 only. I believe it was just because the line the error occurs on changed.